external-crates/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -436,7 +560,7 @@ impl Interpreter {
         data_store: &mut dyn DataStore,
         gas_meter: &mut impl GasMeter,
         extensions: &mut NativeContextExtensions,
-        function: Arc<Function>,
+        function: &Arc<Function>,
```

Comment:
I know this was here but what happens if we pass the reference instead of the `Arc` (or reference to `Arc`?

Reply:
Seems like should be fine. Just needed to update self.call_native_impl and self.call_native_return_values to also take &Function. And entrypoint's call to interpreter.call_native_return_values to pass in &Function. Changes pass all tests